ironfish/src/migrations/data/013-wallet-2/old/accounts.ts

```diff
@@ -5,7 +5,7 @@ import bufio from 'bufio'
 import { IDatabaseEncoding, IDatabaseStore } from '../../../../storage'
 
 const KEY_LENGTH = 32
-const PUBLIC_ADDRESS_LENGTH = 43
+const PUBLIC_ADDRESS_LENGTH = 32
 
```

Comment:
@NullSoldier Do I need to run migration for wallet db for this change during network reset? If I dont need I will keep this change. If I need to run a migration I will not change this file.

Reply:
cc @mat-if

Reply:
You cannot modify old migrations like this or it will break the migration.

Reply:
Ok, I will revert.